### DISCUSSION TITLE
Location of Playlists - Docker volume:ro

### DISCUSSION THREAD

**Discussion by shumancw** [navidrome/navidrome#2410] [Q&A]
I quickly spun up a Navidrome using the compose file. Navidrome is up and functioning, but there is something I don't understand. The compose example has the music folder as read-only. The music folder is where playlists are stored. How are playlists created/updated if they are on a readonly volume?

**Comment by metalheim** [comment]
Navidromes playlists are stored in a database table, the database lives in the data folder.

You can Import (m3u or nsp) playlists from your music folder to navidromes database, but this goes one way only, so no write access needed for the music folder

**Reply by shumancw** [reply]
Thank you!
